<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tempConverterClass</class>
 <widget class="QMainWindow" name="tempConverterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>505</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>温度换算器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupC">
      <property name="title">
       <string>Celsius - 摄氏度</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLCDNumber" name="lcdC"/>
       </item>
       <item>
        <widget class="QDial" name="dialC">
         <property name="minimum">
          <number>-50</number>
         </property>
         <property name="maximum">
          <number>150</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupF">
      <property name="title">
       <string>Fahrenheit - 华氏度</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLCDNumber" name="lcdF"/>
       </item>
       <item>
        <widget class="QDial" name="dialF">
         <property name="minimum">
          <number>-58</number>
         </property>
         <property name="maximum">
          <number>302</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tempconverter.qrc"/>
 </resources>
 <connections/>
</ui>
